src/parseTools.js

```diff
@@ -1117,12 +1117,12 @@ function receiveI64ParamAsI32s(name) {
 function receiveI64ParamAsDouble(name) {
   if (WASM_BIGINT) {
     // Just convert the bigint into a double.
-    return `${name} = Number(${name});`;
```

Comment:
This line could not have worked before.  I believe the reason we didn't see the issue is that so far the only user of this function is wasmfs and I guess we don't test it combination with WASM_BIGINT yet?

Reply:
@tlively does that seem likely?

Reply:
I think that's likely the case, yeah. Fix looks right.